platforms-serverless/lambda/update-code.sh

```diff
@@ -1,10 +1,10 @@
-#!/bin.sh
```

Comment:
Important change? If so, we should also search other files and make sure they all have that.

Reply:
Nothing complained but seemed incorrect

Reply:
Yes, there is another instance o this. Will fix.